### ISSUE TITLE
Linux?

### ISSUE THREAD

**Issue by chrishuges** [pFindStudio/pFind3#3]
Hi,

Thank you for releasing this great tool! I am wondering if you have any plans to transition it to Linux using Mono and the ThermoRawFileParser library? Just the open search tool, not the GUI interface.

Thanks!
Chris

**Comment by Hao-Chi** [comment]
Hi Chris,

Thank you for your mail. Actually, we are preparing for the new version using the ThermoRawFileParser library. The linux version of pFind 3 is also in the plan.

Best Regards,

Hao

**Comment by har-wradim** [comment]
Hi @Hao-Chi,

I see that the current issue has been recently closed. Is the Linux version of the CLI already available to the public?

**Comment by jalew188** [comment]
@har-wradim This may be a mis-operation, we will close this issue after Linux is supported.

**Comment by gsaxena888** [comment]
Any general ETA for the linux version? (I know that the date may slip etc. I was just trying to get a feel for whether we're 1 week, 1 month, 2 months etc away.)

**Comment by dtabb73** [comment]
Yes, I am encountering places on the pFind website that suggest that (older?) Linux versions of this software can be downloaded, but I have not managed to find or form a URL to actually download any Linux version, so far.

**Comment by kfattila** [comment]
Hi, I just second. Will the search engine be available in Linux command line any time soon?
